README.md

```diff
@@ -151,6 +151,20 @@ cli/dist/jiralint search -j "project=MF order by created"
 
 #### Example User Credentials `.jirilintrc`
 
+Note that Cloud Jira access using usernames and passwords is now depreciated. Furthermore, Jira cloud does not allow for custom field creation. When trying to access Jira Cloud using user credentials, use the following:
+
+```json
+{
+    "username": "[username]:",
+    "password": "[API token]",
+    "jiraHost": "agiledigital.atlassian.net",
+    "qualityFieldName": "Quality",
```

Comment:
Are you sure that these shouldn't be `customfield_12345`?

Reply:
they should be - changes have been made